JavaScript/modal.js

```diff
@@ -0,0 +1,134 @@
+      highlight: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+      lang: ['html', 'css', 'javascript'],
+      live: 'https://eserengo.github.io/Portfolio-setup/',
+      source: 'https://github.com/eserengo/Portfolio-setup',
+    },
+    {
+      title: 'Tonic',
+      subtitle: 'CANOPY',
+      techStack: 'Back End Dev',
+      year: 2015,
+      src: './images/Snapshoot-portfolio-3.png',
+      highlight: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+      lang: ['html', 'css', 'javascript'],
+      live: 'https://eserengo.github.io/Portfolio-setup/',
+      source: 'https://github.com/eserengo/Portfolio-setup',
+    },
+    {
+      title: 'Multi Post Stories',
+      subtitle: 'CANOPY',
+      techStack: 'Back End Dev',
+      year: 2015,
+      src: './images/Snapshoot-portfolio-4.png',
+      highlight: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+      lang: ['html', 'css', 'javascript'],
+      live: 'https://eserengo.github.io/Portfolio-setup/',
+      source: 'https://github.com/eserengo/Portfolio-setup',
+    },
+  ];
+
+  const projectCardString = projectsData.map((project) => `
+    <div class="card">
+      <div class="inner-card">
+        <div class="card-img card-1">
+          <img
+            src="${project.src}"
+            alt="Project Snapshoot"
+          />
+        </div>
+        <div class="desc">
+          <h3>${project.title}</h3>
+          <h4>${project.subtitle} <span>${project.techStack}</span> <span>${project.year}</span></h4>
+          <p>${project.highlight}</p>
+          <ul>
+            ${project.lang.map((tech) => `<li>${tech}</li>`).join('')}
+          </ul>
+          <button type="button" class="project-btn button">
+            See Project
+          </button>
```

Comment:
- [ ] [OPTIONAL] I have realized that the design for this section in the new changes that are made are distorted. I recommend that you have a look at it and change it.

**This is how I am seeing it:**

![image](https://user-images.githubusercontent.com/86318284/188919481-706f15fe-a767-479d-a535-805fd7f2e04c.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/188919612-fada1f46-0593-4db1-85b2-da24c0a542ba.png)